addons/crm/crm_lead_menu.xml

```diff
@@ -15,6 +15,7 @@
                 'empty_list_help_model': 'crm.team',
                 'needaction_menu_ref': 'crm.menu_crm_opportunities',
                 'search_default_to_process':1,
+                'search_default_my_leads':1,
```

Comment:
We discussed this with recently, we will not do this move to forcing "my leads". The normal flow is to handle unsassigned and/or new leads, to transform them into opportunities or to discard them. Having a lot of staging leads is not the standard flow we want to implement. I will revert while merging.